.github/workflows/auto-update-collector-version.yml

```diff
@@ -1,27 +0,0 @@
-name: Check collector version
-
-on:
-  workflow_dispatch:
-  schedule:
-    - cron: '0 1 * * *'
-
-jobs:
-  build:
-    # Make sure that the workflow does not run on forks (add yourself as exception if you work on the workflow)
-    if: github.repository_owner == 'open-telemetry' || github.repository_owner == 'svrnm'
```

Comment:
I think that we should have a guard like this in the new workflow. At least the `if: github.repository_owner == 'open-telemetry'` part.

Reply:
this kind of guard used to be helpful, but github disables jobs on (new) forks now, so I haven't been adding them anymore (and you have to remove the guard if you do want to test it on your fork). or is your concern different?

Reply:
Ok, that's good enough for me. Thanks!